coderd/httpmw/authz.go

```diff
@@ -35,3 +36,15 @@ func AsAuthzSystem(mws ...func(http.Handler) http.Handler) func(http.Handler) ht
 		})
 	}
 }
+
+// RecordAuthzChecks enables recording all of the authorization checks that
```

Comment:
```suggestion
// RecordAuthzChecks enables recording all the authorization checks that
```

Reply:
pretty sure it's more correct with the of